templates/add.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
@MuhammadSaadJamil you can use template inheritance here. because the same structure is repeated in list-plan.html and plan-detail.html files.